tmt/steps/provision/__init__.py

```diff
@@ -625,6 +625,11 @@ def is_ready(self) -> bool:
 
         raise NotImplementedError
 
+    @property
+    def logs(self) -> list[str]:
```

Comment:
I'd suggest `log_names` rather than `logs` - `logs` suggests it's really logs, name, content, maybe even more. Since the goal is to return just a list of log names, let's name the property properly.

And please, add a docstring. Check the sources, e.g. https://github.com/teemtee/tmt/blob/main/tmt/steps/provision/__init__.py#L1317, https://github.com/teemtee/tmt/blob/main/tmt/steps/provision/__init__.py#L630 or https://github.com/teemtee/tmt/blob/main/tmt/steps/__init__.py#L1207